crates/subspace-service/src/sync_from_dsn.rs

```diff
@@ -41,6 +41,7 @@ enum NotificationReason {
 
 /// Create node observer that will track node state and send notifications to worker to start sync
 /// from DSN.
+#[allow(clippy::too_many_arguments)] // we don't follow this convention
```

Comment:
We do, but sometimes avoid refactoring things right away to reduce amount of changes. Also again comments at the end of the line rather than before it :confused: